modules/images/webp-uploads/load.php

```diff
@@ -742,3 +742,33 @@ function webp_uploads_get_image_sizes_additional_mime_type_support() {
 
 	return $allowed_sizes;
 }
+
+/**
+ * Updates the quality of images used in the wp editor.
+ * For WP version less than 6.2 returns 82,
+ * Otherwise returns default quality.
```

Comment:
This is not what the goal of this PR is, see #563:
> I propose setting the default quality for WebP images to 82. For versions before 6.1, we should set all mime types to 82, for 6.1+ we can base the quality on mime type and return 82 for WebP

The reason to make the differentiation between 6.1+ and pre-6.1 is the fix in https://core.trac.wordpress.org/changeset/54417 which was part of the 6.1 release: Prior to 6.1, we can't rely on the `$mime_type` filter parameter to be correct.

Reply:
Right, I understood it opposite. I have changed it now.